packages/backend-core/src/environment.ts

```diff
@@ -143,6 +143,7 @@ const environment = {
   POSTHOG_TOKEN: process.env.POSTHOG_TOKEN,
   POSTHOG_PERSONAL_TOKEN: process.env.POSTHOG_PERSONAL_TOKEN,
   POSTHOG_API_HOST: process.env.POSTHOG_API_HOST || "https://us.i.posthog.com",
+  POSTHOG_FEATURE_FLAGS: process.env.POSTHOG_FEATURE_FLAGS,
```

Comment:
NAB: Might be better to call this `POSTHOG_FEATURE_FLAGS_ENABLED` or something since it's a boolean flag